pkg/csi/service/vanilla/controller.go

```diff
@@ -1624,23 +1637,32 @@ func (c *controller) ListVolumes(ctx context.Context, req *csi.ListVolumesReques
 }
 
 func (c *controller) processQueryResultsListVolumes(ctx context.Context, startingToken int, endIndex int,
```

Comment:
processQueryResultsListVolumes is not the function for controller interface.

Reply:
what I mean is that processQueryResultsListVolumes is not a function for controller service. Refer to https://github.com/container-storage-interface/spec/blob/master/csi.proto#L62-L101

Since this was implemented like this previously, let us refactor this later.

Reply:
processQueryResultsListVolumes function is implemented as method for object of type 'controller', which is defined locally and not of 'Controller' service
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/csi/service/vanilla/controller.go#L69-L80